hivemq-edge/src/frontend/src/api/hooks/useFrontendServices/useGetNotifications.tsx

```diff
@@ -6,8 +6,11 @@ import { ApiError, NotificationList } from '@/api/__generated__'
 export const useGetNotifications = () => {
   const appClient = useHttpClient()
 
-  return useQuery<NotificationList, ApiError>([QUERY_KEYS.FRONTEND_NOTIFICATION], async () => {
-    const item = await appClient.frontend.getNotifications()
-    return item
+  return useQuery<NotificationList, ApiError>({
+    queryKey: [QUERY_KEYS.FRONTEND_NOTIFICATION],
+    queryFn: async () => {
+      const item = await appClient.frontend.getNotifications()
+      return item
```

Comment:
suggestion(non-blocking): can be returned directly and aligns with other implementation within the PR

```suggestion
      return await appClient.frontend.getNotifications()
```